Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(jenkinsVersions: [null, '2.32.3', '2.47'])
+buildPlugin(jenkinsVersions: [null, '2.32.3', '2.60.3', '2.73.1'])
```

Comment:
Probably you want to bump up the `jenkins.version` to 2.7.3, in which case we could drop the tests against 2.32.x I think.